lib/processors/background/BackgroundProcessor.ts

```diff
@@ -295,45 +337,34 @@ export abstract class BackgroundProcessor extends Processor {
 
   protected abstract _getWebGL2PipelineType(): WebGL2PipelineType;
 
-  protected abstract _setBackground(inputFrame: OffscreenCanvas | HTMLCanvasElement | HTMLVideoElement): void;
```

Comment:
HTMLVideoElement is one of the possible inputs right? Is this a breaking change?

Reply:
This function is for internal use only. `processFrame()` still accepts an HTMLVideoElement.